src/js/vre/chapter31/containers/ConfirmationPage.jsx

```diff
@@ -8,11 +22,75 @@ class ConfirmationPage extends React.Component {
   }
 
   componentDidMount() {
+    focusElement('.confirmation-page-title');
+    scrollToTop();
+  }
+
+  toggleExpanded = (e) => {
+    e.preventDefault();
+    this.setState({ isExpanded: !this.state.isExpanded });
   }
 
   render() {
+    const form = this.props.form;
+    const response = form.submission.response
+      ? form.submission.response.attributes
+      : {};
+    const formData = this.props.form.data || {};
+
+    const name = formData['view:isVeteran']
```

Comment:
I don't think we have this field in chapter 31?

Reply:
Thanks, you're correct. Sorry about the copy pasta, will fix.